var/spack/repos/builtin/packages/arborx/package.py

```diff
@@ -69,3 +69,37 @@ def cmake_args(self):
                 '-DCMAKE_CXX_COMPILER=%s' % spec["kokkos"].kokkos_cxx)
 
         return options
+
+    examples_src_dir = "examples"
+
+    @run_after('install')
+    def setup_build_tests(self):
+        """Copy the example source files after the package is installed to an
+        install test subdirectory for use during `spack test run`."""
+        self.cache_extra_test_sources([self.examples_src_dir])
+
+    def build_tests(self):
+        """Build test."""
+        cmake_build_path = join_path(self.install_test_root, self.examples_src_dir, "build")
+        mkdirp(cmake_build_path)
+        cmake_prefix_path = "-DCMAKE_PREFIX_PATH={0};{1}".format(self.spec['arborx'].prefix)
+
+        if '+mpi' in self.spec:
```

Comment:
I see you removed `spec['kokkos']`. That's fine, but please add a comment why it is not necessary, so that a reader not familiar with ArborX can understand that.